### ISSUE TITLE
Bonus Completist Prize

### ISSUE THREAD

**Issue by obscurerichard** [freezingsaddles/freezing-web#231]
x

**Comment by arichnad** [comment]
Something a bit strange about the bonus points between 9 and 10 miles.

Points looks normal:

![Image](https://github.com/user-attachments/assets/f40d89ce-5a6e-47b6-a5f7-3dc18f708d35)

It is curious, 55 isn't the correct maximum?  It's 55.125? At 9.5 miles?  After 10 miles the maximum bonus points is 55!

![Image](https://github.com/user-attachments/assets/4a98aae8-3ce2-4030-8006-7bd7c4115f5d)

**Comment by merlinorg** [comment]
@arichnad yes, those are the worst .5 miles you can ride in the competition.
